src/components/Download/Quiz.vue

```diff
@@ -65,13 +65,41 @@
               <a :href="downloads.nukkit" v-if="options.nukkit" download>
                 Download
               </a>
+			        <a :href="downloads.bungee" v-if="options.bungee" download>
+			          Download
+			        </a>
+			        <a :href="downloads.velocity" v-if="options.velocity" download>
+			          Download
+			        </a>
             </div>
           </template>
+		      <template v-if="options.legacy">
+			      <img alt="LuckPerms logo" src="@/assets/logo.png">
+			      <h1>You need LuckPerms{{ legacy }} for {{ serverType }}</h1>
+			      <div class="options">
+			        <a :href="downloads.bukkit-legacy" v-if="options.bukkit" download>
```

Comment:
Note, I'm speaking about this line. Here `downloads.bukkit-legacy` doesn't work.

Reply:
It needs to be `downloads['bukkit-legacy']` otherwise it's trying to perform arithmetic on an object.